applications/ConstitutiveLawsApplication/tests/HighCycleFatigue/high_cycle_fatigue_test_parameters.json

```diff
@@ -0,0 +1,119 @@
+        "model_part_name"                    : "Structure",
+        "domain_size"                        : 3,
+        "analysis_type"                      : "non_linear",
+        "model_import_settings"              : {
+            "input_type"     : "mdpa",
+            "input_filename" : "HighCycleFatigue/high_cycle_fatigue_test"
+        },
+        "material_import_settings"           : {
+            "materials_filename" : "HighCycleFatigue/high_cycle_fatigue_test_materials.json"
+        },
+        "time_stepping"                      : {
+            "time_step" : 1.0
+        },
+		"use_old_stiffness_in_first_iteration": true,
+        "line_search"                        : false,
+        "convergence_criterion"              : "residual_criterion",
+        "displacement_relative_tolerance"    : 0.0001,
+        "displacement_absolute_tolerance"    : 1e-9,
+        "residual_relative_tolerance"        : 0.0001,
+        "residual_absolute_tolerance"        : 1e-9,
+        "max_iteration"                      : 10,
+        "rotation_dofs"                      : false
+    },
+    "processes" : {
+        "constraints_process_list" : [{
+            "python_module" : "assign_vector_variable_process",
+            "kratos_module" : "KratosMultiphysics",
+            "Parameters"    : {
+                "model_part_name" : "Structure.DISPLACEMENT_Displacement_Auto1",
+                "variable_name"   : "DISPLACEMENT",
+                "constrained"     : [true,true,true],
+                "value"           : [0.0,0.0,0.0],
+                "interval"        : [0.0,"End"]
+            }
+        },{
+            "python_module" : "assign_vector_variable_process",
+            "kratos_module" : "KratosMultiphysics",
+            "Parameters"    : {
+                "model_part_name" : "Structure.DISPLACEMENT_Displacement_Auto2",
+                "variable_name"   : "DISPLACEMENT",
+                "constrained"     : [true,true,true],
+                "value"           : [0.0,0.0,"-(2.85E-03)*cos(pi*(t+2)/6.0)+(2.85E-03)"],
+                "interval"        : [0.0,"End"]
+            }
+        }],
+        "json_check_process" : [
+        {
+            "python_module"   : "from_json_check_result_process",
+            "kratos_module" : "KratosMultiphysics",
+            "help"                  : "",
+            "process_name"          : "FromJsonCheckResultProcess",
+            "Parameters"            : {
+                "gauss_points_check_variables" : ["DAMAGE","UNIAXIAL_STRESS"],
+                "input_file_name"              : "HighCycleFatigue/high_cycle_fatigue_test_results.json",
+                "model_part_name"              : "Structure.Parts_Parts_Auto1",
+                "time_frequency"               : 3.0
+            }
+        }
+        ],
+        "loads_process_list"       : [],
+        "list_other_processes"     : []
+    },
+    "_json_output_process" : [

```

Comment:
FYI now our json supports comments, maybe this would be more clear
=> same syntax as comments in C++

Reply:
> FYI now our json supports comments, maybe this would be more clear
> => same syntax as comments in C++

OK! I will comment these lines then.